content/pt-br/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,114 @@
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) habilitado
+(o recurso está ativo por padrão desde o Kubernetes v1.29), você pode especificar uma `restartPolicy`
+para contêiners listados no campo `initContainers` de um Pod.
+Esses _sidecar_ contêiners reiniciáveis são independentes de outros init containers 
+e dos contêineres principais da aplicação dentro do mesmo pod. Eles podem ser iniciados, interrompidos ou reiniciados sem afetar o contêiner principal da aplicação e outros init containers.
+
+Você também pode executar um Pod com vários contêineres que não estão marcados como init ou sidecar containers. Isso é apropriado se os contêineres dentro do Pod forem necessários para que o Pod funcione no geral, mas você não precisa controlar quais contêineres iniciam ou param primeiro.
+Você também pode fazer isso se precisar dar suporte a versões mais antigas do Kubernetes que não suportam um campo `restartPolicy` em nível contêiner.
+
+### Aplicação de exemplo {#sidecar-example} 
+
+Aqui está um exemplo de uma implantação com dois contêineres, um dos quais é um 
+sidecar:
+
+{{% code_sample language="yaml" file="application/deployment-sidecar.yaml" %}}
+
+## Sidecar containers e ciclo de vida do Pod 
+
+Se um init container  for criado com seu `restartPolicy` definido como `Always`, ele
+iniciará e permanecerá em execução durante toda a vida útil do Pod. Isso pode ser útil para executar serviços de suporte separados dos contêineres principais da aplicação.
+
+Se um `readinessProbe` for especificado para este init container, seu resultado será usado para determinar o estado `ready` do Pod.
+
+Como esses contêineres são definidos como init containers, eles se beneficiam da mesma
+ordem e garantias sequenciais que os regular init containers, permitindo que você misture sidecar contêineres com regular init containers para fluxos complexos de inicialização de Pod.
+
+Comparados aos regular init containers, os sidecars definidos em `initContainers` continuam sendo executados após terem sido iniciados. Isso é importante quando há mais
+de uma entrada dentro de `.spec.initContainers` para um Pod. Depois que um sidecar-style init container está em execução (o kubelet
+definiu o status `started` para esse init container como true), o kubelet então inicia o próximo init container da lista ordenada em `.spec.initContainers`.
+Esse status se torna verdadeiro porque há um processo em execução no
+contêiner e nenhuma sonda de inicialização definida, ou como resultado do sucesso de seu `startupProbe`.
+
+### Jobs com sidecar containers
+
+Se você definir um Job que usa sidecar usando Kubernetes-style init containers,
+o sidecar container em cada Pod não impede que o Job seja concluído após o
+contêiner principal ser terminado.
+
+Aqui está um exemplo de um Job com dois contêineres, um dos quais é um sidecar:
+
+{{% code_sample language="yaml" file="application/job/job-sidecar.yaml" %}}
```

Comment:
It should have been search for job-sidecar.yaml at /en/examples/application/job/ but instead it search at /en/concepts/application/job which does not exist, Thats why there is an error.

Reply:
I appreciate your help.